internal/network/command.go

```diff
@@ -188,6 +190,11 @@ func printHumanTable(cmd *cobra.Command, network networkingv1.NetworkingV1Networ
 		}
 	}
 
+	if network.Status.GetIdleSince() != nil {
```

Comment:
Why wouldn't the backend always return this value?

Reply:
A network can be either idle or active. The idle_since timestamp only exists if the network is idle.